__tests__/app.test.js

```diff
@@ -47,3 +47,41 @@ describe("GET /api/topics", () => {
       });
   });
 })
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an article with the corresponding id", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.article).toEqual(
+        expect.objectContaining({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 100,
+          article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        })
+      )
+    })
```

Comment:
Good test here, well done on being specific and just checking a string for created at is fine!